packages/files-ui/src/Components/Modules/FileBrowsers/MoveFileModal.tsx

```diff
@@ -196,17 +221,16 @@ const MoveFileModule = ({ filesToMove, modalOpen, onClose, onCancel, mode }: IMo
         alignItems="center"
         className={classes.paddedContainer}
       >
-        {mode === "move" && movePath === currentPath && (
+        {!!movePath && !isAllowedToMove && (
           <Typography
             component="p"
             variant="body1"
           >
-            <Trans>
-              {plural(filesToMove.length, {
-                one: "The file is already in this folder",
-                other: "The files are already in this folder"
-              })}
-            </Trans>
+            {
+              folders.length
```

Comment:
The right error comes up this way
```suggestion
              movePath !== currentPath
```

Reply:
I don't think the error is wrong, in which case do you have it wrong? To have better error, we should check whether we are talking about folders, files, or mixed. In that case, I'm checking whether there's at least one folder selected, in which case I talk about folders, otherwise about files. I think the error are half wrong then.

Reply:
You're right, this change is not needed as I think about it now,
I was going for `The files are already in this folder`, when you have folders selected,
but `You can't move into this folder` works for me.

Reply:
Now i feel whats going on here, And the change is not needed.
I was going for `The files are already in this folder` even for folder,
but `You can't move into this folder` works just fine!